frontend-react/src/content/internal-user-guides/make-markdown-pages.md

```diff
@@ -0,0 +1,84 @@
+# Make Markdown Pages
+
+Building static pages from markdown is a simple process. First, you create 
+your markdown file, then you create a `MarkdownDirectory` by providing a 
+label (seen in navigation), slug (url extension), and array of imported 
+markdown files.
```

Comment:
since the `BuiltForYou` page is there as a great example, maybe would be good to explicitly link to it for reference here

Reply:
We could! Pretty sure we'd have to link to the GH url tho, not the path, since this renders on the site for admins.